packages/amplify_authenticator/assets/Inter/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2020 The Inter Project Authors (https://github.com/rsms/inter)

```

Comment:
nit: The license in their GH has this line: 
`"Inter" is trademark of Rasmus Andersson.`
Not sure if that's important?

Reply:
It's the license file they provide